plugins/drivers/proto/driver.proto

```diff
@@ -448,14 +448,17 @@ enum TaskState {
 // TaskHandle is created when starting a task and is used to recover task
 message TaskHandle {
 
+    // Version is used by the driver to version the DriverState schema
+    int32 version = 1;
```

Comment:
Validate that the driver doesn't return a task handle with value of 0?

Reply:
Its actually ok if a third party driver does this. I added a comment that version 0 is reserved and shouldn't be used. It feels wierd to validate that after you already started a task your returning a positive version number.